### PR TITLE
Remove obsolete attribute 'name' from PLR

### LINKED ISSUE #1265: Remove all name attributes in yml plr parts
The numbering of PLR's is a old left over. It can be removed to clean and simplify code.

This is mostly related to the plr parts in the yml config:
https://github.com/openoereb/pyramid_oereb/blob/master/pyramid_oereb/standard/pyramid_oereb.yml.mako#L449

All related code should be cleaned an refactored. Also the test data can be moved to get rid of the plr119 folder:
https://github.com/openoereb/pyramid_oereb/tree/master/sample_data/plr119

### PR DESCRIPTION
**Pull request by vuilleumierc**
Closes #1265

### CODE REVIEW

File: pyramid_oereb/standard/pyramid_oereb.yml.mako [hunk 1]
```diff
@@ -919,8 +907,7 @@ pyramid_oereb:
       #        - SubTheme2
```
Comment (voisardf) [inline pyramid_oereb/standard/pyramid_oereb.yml.mako:907]: we could remove this too, as there ar no subthemes in this topic or move the comments to the LandUsePlan where SubThemes are allowed

**Review by voisardf — COMMENTED** [review]
Looks good to me, one comment that is not directly related to the issue, but could be done while you are at it

**Review by vvmruder — CHANGES_REQUESTED** [review]
There are still some PLRxyz left in docker/config.yaml.tmpl e.g.: https://github.com/openoereb/pyramid_oereb/blob/1265-remove-plr-attribute-name/docker/config.yaml.tmpl#L448

**Comment by vuilleumierc** [comment]
> There are still some PLRxyz left in docker/config.yaml.tmpl e.g.: https://github.com/openoereb/pyramid_oereb/blob/1265-remove-plr-attribute-name/docker/config.yaml.tmpl#L448

Oh sorry, I thought we wouldn't be using this anymore since the new standard dev setup runs in Docker. I never updated it in my latest PR

**Review by vvmruder — APPROVED** [review]
Sorry for late reply. Yes-feel free to merge.

**Pull request merged**